frontend/src/components/Map/Location.tsx

```diff
@@ -91,7 +91,7 @@ export class Location extends React.Component<Location.Props, Location.State> {
       city,
     } = this.props.node;
 
-    let validatorRow = null;
+    let validatorRow = <div />;
```

Comment:
Any reason for this change? This will insert an empty `div` inside a `tbody`, which doesn't quite make sense.

Reply:
This comes from activating more null check rules.